README.md

```diff
@@ -157,6 +157,45 @@ Out[] = {{1, 2, 5, 3, 6}, {5, 3, 6, 3}, {6, 3, 8}, {8, 9}, {17}}
 
 All of these functions have `Method`, `TimeConstraint` and `"EventOrderingFunction"` options. `TimeConstraint` is self-evident, the other two work the same way as they do in `WolframModel` and will be described further in the `WolframModel` part of this README.
 
+## ToPatternRules
+
+`ToPatternRules` is a convenience function used to quickly enter rules such as the one mentioned previously
```

Comment:
I am finding it hard to understand what this is doing. Can you explain to me what this example is doing and perhaps I will have an idea what should be explained here?
In[] := ToPatternRules[{{v1, v2, v3}, {v2, v4, v5}} ->
  {{v5, v6, v1}, {v6, v4, v2}, {v4, v5, v3}}]
Out[] = {{v1_, v2_, v3_}, {v2_, v4_, v5_}} :>
 Module[{v6}, {{v5, v6, v1}, {v6, v4, v2}, {v4, v5, v3}}]

Reply:
Well, `{{v1, v2, v3}, {v2, v4, v5}} -> {{v5, v6, v1}, {v6, v4, v2}, {v4, v5, v3}}` is the rule written in the ordinary `WolframModel` format (just with `v1`, `v2`, ... used instead of `1`, `2`, `3`, ...). It is converted to the pattern rule `{{v1_, v2_, v3_}, {v2_, v4_, v5_}} :> Module[{v6}, {{v5, v6, v1}, {v6, v4, v2}, {v4, v5, v3}}`, which is (almost) the kind of rule `SubsetReplace` is using, and is similar to normal `Replace` rules.

The second, pattern-rule format with the `Module`, is in a sense a "low-level", "power-user" format that allows for a lot of freedom (like choosing your own name for the new vertices, custom nesting, custom conditions on the vertices, etc.).

The first anonymous-rule format (which is equivalent to `{{1, 2, 3}, {2, 4, 5}} -> {{5, 6, 1}, {6, 4, 2}, {4, 5, 3}}`) is a simplified format, which is easier to read and enter, but does not allow for as much customization.